das_client/app/lib/pages/journey/journey_table/widgets/header/header_box.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:app/pages/journey/journey_table/header/radio_channel/radio_channel_view_model.dart';
+import 'package:app/pages/journey/journey_table/journey_position/journey_position_view_model.dart';
+import 'package:app/pages/journey/journey_table/widgets/detail_modal/detail_modal_view_model.dart';
+import 'package:app/pages/journey/journey_table/widgets/header/journey_identifier.dart';
+import 'package:app/pages/journey/journey_table/widgets/header/radio_channel.dart';
+import 'package:app/pages/journey/journey_table_view_model.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:sbb_design_system_mobile/sbb_design_system_mobile.dart';
+import 'package:skeletonizer/skeletonizer.dart';
+
+const double _width = 215.0;
+const double _smallWidth = 150.0;
+const double _height = 112.0;
+
+class HeaderBox extends StatelessWidget {
```

Comment:
Nitpick: `HeaderBox` ist sehr generell. Evtl. `LeftHeaderBox` oder `JourneyIdentifierHeaderBox` (RadioChannels fehlen..). Bin mir aber auch nicht sicher, wie man die am Besten nennt 😄  Siehe auch Kommentar in `MainContainer`.